src/language/githubReleaseInstaller.ts

```diff
@@ -73,7 +73,6 @@ export class GitHubReleaseInstaller {
     } catch(error: unknown) {
       this.writeStatus('Standalone language server installation failed:');
       this.writeStatus(`> ${error}`);
-      console.error('dafny installation failed', error);
```

Comment:
I think this is a good summary of handling any error. Could you put it back please or justify here why you removed it?

Reply:
It ends up being printed while running the tests which is noisy. Is it important to print to the console given that it's also printed to the output channel?

Reply:
I hadn't realized it was console.error(). I guess it's fine then since it's not displayed to the user.